app/server/src/main/scala/org/bitcoins/server/Main.scala

```diff
@@ -22,7 +23,14 @@ import scala.concurrent.duration._
 import scala.concurrent.{Await, Future}
 
 object Main extends App {
-  implicit val conf = BitcoinSAppConfig.fromDefaultDatadir()
+  implicit val conf = {
+    val path: Path = if (args.isEmpty) {
```

Comment:
this is less than ideal as we add args this doesn't really scale (i.e. we can't omit the `datadir` arg)